index.js

```diff
@@ -186,30 +186,31 @@ module.exports = class Hyperdrive extends ReadyResource {
   }
 
   async del (name) {
-    if (!this.opened) await this.ready()
```

Comment:
I think it should be added everywhere, but the simpler and current consistent choice is to remove it